Apps/RaptureIntTests/src/test/java/rapture/structured/StructuredApiIntegrationTests.java

```diff
@@ -86,7 +87,7 @@
      */
     @BeforeClass(groups = { "nightly", "search" })
     @Parameters({ "RaptureURL", "RaptureUser", "RapturePassword" })
-    public void setUp(@Optional("http://localhost:8665/rapture") String url, @Optional("rapture") String username, @Optional("rapture") String password) {
+    public void setUp(@Optional("http://192.168.99.100:8665/rapture") String url, @Optional("rapture") String username, @Optional("rapture") String password) {
```

Comment:
I can change it back to localhost. These days I test against docker rather than local instances.